pkg/genericapiserver/config.go

```diff
@@ -318,16 +313,13 @@ func (c Config) New() (*GenericAPIServer, error) {
 	}
 
 	if c.RestfulContainer != nil {
-		s.mux = c.RestfulContainer.ServeMux
```

Comment:
If `c.RestfulContainer` is set, it is used for `s.HandlerContainer`. Further down `s.Mux` is set to the `ServeMux` of the container. So it's not ignored. Any use case I am missing here?